kickstarter/android-oss
# WEB-827: Add Page Viewed analytic to Creator Dashboard

# 📲 What

Create a segment page viewed event for the Creator Dashboard.

# 🤔 Why

The native audit initiative

# 👀 See

![Screen Shot 2022-12-19 at 6 45 20 PM](https://user-images.githubusercontent.com/19390326/208549481-a6f4e467-73b0-4d2e-839c-b19f8af3ef95.png)

# 📋 QA

- Log into creator account
-  View creator dashboard
- Check segment dashboard (note that you have to be on staging to test the analytic events on segment)

# Story 📖

[WEB-827: Add Page Viewed analytic to Creator Dashboard](https://kickstarter.atlassian.net/browse/WEB-827)
